src/helpers/PdfDownload/index.js

```diff
@@ -0,0 +1,38 @@
+export class PdfDownload {
+  constructor(event, form) {
+    this.submitEvt = event;
+    this.formInstance = form;
+  }
+
+  /**
+   * isPdfDownloadEnabled function check needPdf field is marked as 'yes' in form.io
+   * and then set PDF url as a session storage
+   * @return {undefined}
+   * */
+  isPdfDownloadEnabled() {
+    if (this.submitEvt.data && this.submitEvt.data.needPdf === "yes") {
+      const pdfUrl = this.submitEvt.metadata.pdfUrl.DownloadUrl;
+      window.sessionStorage.setItem("pdfUrl", pdfUrl);
+      this.formInstance.root.nextPage().then(() => {
+        document.getElementsByClassName(
```

Comment:
why do you want to hide the nav container here? as this can be controlled in thewizard form page settings in form.io

Reply:
There is no setting to hide submit button, there is a config to hide "breadcrumb"


usually, the apps with multiple wizards hide that submit button logically in their app so that should not be an issue. if that appear we hide that